packages/geo/src/lib/layer/layer-list/layer-list-binding.directive.ts

```diff
@@ -26,20 +29,44 @@ export class LayerListBindingDirective implements OnInit, AfterViewInit, OnDestr
   ngOnInit() {
     // Override input layers
     this.component.layers = [];
-
-    this.layers$$ = this.mapService
-      .getMap()
-      .layers$.subscribe((layers: Layer[]) => {
-        this.component.layers = layers.filter((layer: Layer) => {
-          return layer.showInLayerList === true;
-        });
+    this.layersOrResolutionChange$$ = combineLatest([
```

Comment:
This works but is not ideal. See the last comment of this review. Also, settting the layers eveytime the resolution changes is not needed.

Reply:
Actually, does it work? isInResolutionsRange$ is never updated unless it's getter is invoked and I don't see that anywhere.